Resources/doc/2-the-view-layer.rst

```diff
@@ -88,10 +94,12 @@ or it is possible to use lazy-loading:
 
     <?php
 
-    use FOS\RestBundle\Controller\FOSRestController;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-    class UsersController extends FOSRestController
+    class UsersController extends Controller
     {
+        use ContainerAwareControllerTrait;
```

Comment:
```Symfony\Component\DependencyInjection\ContainerAware and FOS\RestBundle\Controller\ContainerAwareControllerTrait define the same property ($container) in the composition of FOS\RestBundle\Tests\Fixtures\Controller\AnnotatedNonPluralizedArticleController. This might be incompatible, to improve maintainability consider using accessor methods in traits instead. Class was composed```